tests/unit/test_charm.py

```diff
@@ -541,6 +541,9 @@ def test_enable_disable_extensions(harness, caplog):
   plugin_timescaledb_enable:
     default: false
     type: boolean
+  plugin_audit_enable:
+    default: false
```

Comment:
We are testing possibility to enable pgaudit.
IMHO, we should keep default true, but test possibility of disabling it in the test below.

Reply:
Good catch! Fixed on https://github.com/canonical/postgresql-operator/pull/612/commits/e1d291686f486dc8bb5a3eea2b9dac1ad6f831c7.